index.bs

```diff
@@ -2931,8 +2933,12 @@ partial interface MLGraphBuilder {
     if (options.bTranspose)
       b = builder.transpose(b);
 
-    let ab = builder.matmul(builder.mul(builder.constant(options.alpha), a), b);
-    return (c ? builder.add(ab, builder.mul(builder.constant(options.beta), c)) : ab);
+    let ab = builder.matmul(
+        builder.mul(builder.constant(a.datatType, options.alpha), a), b);
```

Comment:
typo `a.datatType`

Reply:
thanks - fixed in 68eafc28efb22d2a15d7794b2505b64c1b28f2a5